### DISCUSSION TITLE
Sumber Database

### DISCUSSION THREAD

**Discussion by SKumulia** [teamalgoritma/community#15] [Q&A] [answered]
Darimana biasa nya sumber database untuk dipakai di Jupyter Notebook? Apakah umumnya selalu dari Ms Excel?

**Comment by inytss** [comment, marked as answer]
Untuk sumber data bisa dari mana saja, bisa berupa file excel, csv, database SQl maupun Non SQL dapat diolah dalam python menggunakan jupyter notebook.

**Reply by SKumulia** [reply]
Apakah yg diambil hanya data saja? Karena kadang dari Ms Excel biasa ada formula nya juga. Dan kalau diambil apa implikasinya ke saat di pakai di Jupyter Notebook/Python?

Dan apakah data2 dari program lain dapat langsung dibaca oleh Jupyter Notebook/Python atau perlu untuk import extension?

**Reply by muhrisman** [reply]
Untuk yang terambil hanya datanya saja pak, misal kolom ketiga adalah penjumlahan dari kolom pertama dan kolom kedua maka yang terbaca hanyalah datanya (misal nilai numerik). Untuk data dengan tipe lain misal data dalam bentuk peta (shp) dapat pula diimport ke dalam Jupyter Notebook. Mengenai extension diperlukan juga untuk mengimport ekstensi tertentu misal import geopandas untuk data peta.

**Reply by inytss** [reply]
Apabila file data yang diimport dalam bentuk excel memang hanya data saja, formula yang ada didalamnya tidak akan diekstrak atau tersimpan ketika dibaca pada python/Jupyter notebook. 
Untuk data dengan ekstensi yang lain ada kemungkinan kita memerlukan package lain untuk mengimport datanya atau bisa juga masih cukup menggunakan package `pandas`, contoh untuk data dengan ekstensi excel bisa menggunakan `pandas`.

**Comment by muhrisman** [comment]
Untuk sumber data dapat berasal dari berbagai macam sumber dengan ekstensi file yang berbeda-beda. Sumber tersebut dapat diolah menggunakan Python dengan jupyter notebook.
